apps/remix-ide/src/app.js

```diff
@@ -387,7 +387,8 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
     debug,
     analysis,
     test,
-    filePanel.remixdHandle
+    filePanel.remixdHandle,
+    filePanel.gitHandle
```

Comment:
as a Plugin I would recommend putting Git at the root of the project and access its methods only through remix-plugin API

Reply:
will plan this changes later on. This will anyway go in during the react refactoring.